receiver/vcenterreceiver/metadata.yaml

```diff
@@ -73,6 +73,10 @@ attributes:
     enum:
       - virtual
       - physical
+  cpu_capacity_state:
```

Comment:
"state" might not be the right word? if you want to keep this a bool, i think "cpu_capacity_reserved" or probably even just "cpu_reserved" would be better. Description would change to "Whether the CPU is reserved." in the case of the latter.

Reply:
Actually no matter what I'd add to the description "... is reserved for virtual machines."

Reply:
Can resolve